utilities/update_docs_version.sh

```diff
@@ -10,7 +10,7 @@ RELEASED_VERSION=$(mvn org.apache.maven.plugins:maven-help-plugin:2.1.1:evaluate
 if [ "${RELEASED_VERSION##*-}" != "SNAPSHOT" ]; then
     echo "Changing version to $RELEASED_VERSION in README files"
     # Get list of directories for which README.md must be updated
-    module_folders=($(find . -maxdepth 1 -name 'gcloud-java*' -type d) .)
+    module_folders=($(find . -maxdepth 2 \( -regex "./gcloud-java[-[a-z]+]+*" -o -regex "./gcloud-java-contrib/gcloud-java[-[a-z]+]+*" \) -type d) . ./gcloud-java)
```

Comment:
Try `echo ${module_folders[*]}` for the second line - does that give you all the results?